### PR TITLE
update store menu to include pt-br

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Updates the store journey menu with the pt-br guides.
Includes "Retrieve database info" guide.

### CODE REVIEW

File: src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Como criar uma tabela no Edge SQL
 description: >-
   Descubra instruções passo a passo para criar uma tabela em um banco de dados Edge SQL para gerenciamento eficiente de dados.
-permalink: /documentation/products/store/sql/create-table/
+permalink: /documentation/products/store/sql/criar-tabela/
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx:5]:
```suggestion
permalink: /documentacao/produtos/store/sql/criar-tabela/
```

**Review by gabriel-azion — APPROVED** [review]
be certain you ran build locally, other than that LGTM

File: src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 meta_tags: >-
   Edge SQL, SQL database, create table, database management, CRUD operations, API integration, data storage, SQL tutorial, Edge SQL guide, database creation, SQL commands
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx:7]:
```suggestion
  Edge SQL, banco de dados SQL, criar tabela, gerenciamento de banco de dados, operações CRUD, integração API, data storage, SQL tutorial, guia Edge SQL, criação de banco de dados, comandos SQL
```

File: src/i18n/pt-br/storeMenu.ts [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+			{
+				text: 'Crie um banco de dados',
+				header: true,
+				anchor: true,
+				type: 'learn',
+				slug: '/documentacao/produtos/store/sql/criar-banco-de-dados/',
+				key: 'createDatabase',
+			},
+			{
+				text: 'Liste os banco de dados',
```
Comment (MarianaAguilera) [inline src/i18n/pt-br/storeMenu.ts:161]:
```suggestion
				text: 'Liste os bancos de dados',
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by robsongajunior — APPROVED** [review]

File: src/content/docs/pt-br/pages/store-jornada/sql/recuperar-informacoes-banco.mdx [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Como recuperar informações de um banco de dados do Edge SQL
 description: >-
   Aprenda a recuperar informações de um banco de dados do Edge SQL.
-permalink: /documentacao/produtos/store/sql/recuperar-info-do-banco/
+permalink: /documentacao/produtos/store/sql/recuperar-informacoes-do-banco/
```
[the remark this answers is not in the document]
Comment (robsongajunior) [reply src/content/docs/pt-br/pages/store-jornada/sql/recuperar-informacoes-banco.mdx:5]: validamos, não tem nenhum link nas páginas para essa doc

**Pull request merged**